contents/docs/error-tracking/managing-issues.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Issue management
+---
+
+Our error tracking product gives you multiple ways to help you manage issues and work towards resolution. 
+
+## Assigning issues
+
+Issues can be assigned to a teammate from the list or issue page. To do this, select the **assignee** option and search for the teammate you want to assign it to, and select them. You can click this dropdown again to remove the assignee.
+
+What if you want to assign issues to a group rather than an individual teammate? You can create a role [your project settings](https://us.posthog.com/settings/organization-roles) under **Error tracking**.
```

Comment:
Group is a bit loaded with other context in PostHog language
```suggestion
Want to assign issues to a **team** rather than an individual teammate? You can create a role [your project settings](https://us.posthog.com/settings/organization-roles) under **Error tracking**.
```